### ISSUE TITLE
Add a keras.io example using KerasNLP!

### ISSUE THREAD

**Issue by mattdangerw** [keras-team/keras-hub#189]
KerasNLP is always looking for new examples on [keras.io](https://keras.io/keras_nlp) the demonstrate how to use the library.

This issue will stay open on a "contributions welcome" list forever. If you have an idea for a new example, feel free to leave a comment here to discuss, and then we can split out an issue to track the specific example once we have consensus.

**Comment by shubranshugupta** [comment]
Hey! I am an absolute beginner, I would like to contribute to this project. Also, suggest some relevant resources.

**Comment by Plutone11011** [comment]
Hi, is contribution to this issue still welcomed?

**Comment by Plutone11011** [comment]
Hi @mattdangerw, do you think an example of Text Summarization might be interesting? It's also in the keras.io list of examples.

**Comment by Neeshamraghav012** [comment]
what about semantic search using KerasNLP @mattdangerw?

**Comment by jayam30** [comment]
Hey, I want to begin my contribution in Keras with this good first issue, please assign me

**Comment by shivance** [comment]
[#1253](https://github.com/keras-team/keras-io/pull/1253) Semantic Similarity using KerasNLP

**Comment by atharvapurdue** [comment]
Hi  @mattdangerw ,
I had 2 examples which I feel might be a good example to contribute to keras.io 
Example  1 -
BERT model for Chemical Reaction Yield Prediction (Regression Task)- (reference - https://iopscience.iop.org/article/10.1088/2632-2153/abc81d ). The dataset which will be used is this - https://rxn4chemistry.github.io/rxn_yields/data/ 
I will make sure that the tutorial is easy to follow even for a non-domain expert 
Example 2 - 
Text Summerization using BART and Sampler functions - Dataset which will be used - https://www.kaggle.com/datasets/gowrishankarp/newspaper-text-summarization-cnn-dailymail 

I am hoping that I can contribute more to this community. Please let me know if this is something that the community would like on keras.io

**Comment by taham655** [comment]
Hi, is there any good first issue that I can be assigned to?

**Comment by amangupta201** [comment]
@mattdangerw 
A Simple_MNIST_Covnet can also be a good idea
https://github.com/amangupta201/keras-nlp/blob/master/Simple_MNIST_convnet.ipynb

**Comment by MrunaliG** [comment]
Hi, I would like to contribute an example that's a good first issue here. Would migrating the keras V2 example to V3 for Large-scale multi-label text classification https://keras.io/examples/nlp/multi_label_classification/ fall under this?

**Comment by mattdangerw** [comment]
@MrunaliG

**Comment by arpitsinghgautam** [comment]
Hi, I would like to contribute an example to the Keras documentation that's a good first issue here. Specifically, I propose creating a tutorial on fine-tuning LLM (Gemma 1b I was thinking) using Group Relative Policy Optimization (GRPO) following the approach outlined in the DeepSeek paper.

**Comment by rashmivemuri** [comment]
Hi everyone, I’m Rashmi, a BTech AI first-year student, and I’m new to open-source contributions.
I want to start contributing to Keras and learn the proper contributor workflow. I’m comfortable with Python but I don't have much exposure to machine learning, and I’ve never contributed to an open-source project before.
I’d really appreciate guidance on:
where an absolute beginner should start with Keras,
whether there are any good-first issues or beginner-friendly tasks.

**Comment by maitry63** [comment]
@rashmivemuri , Thank you for your interest. 
We have many beginner friendly issues for contribution. You can take a look into the "contributions welcome" labeled issues in the repository.
